tools/cli/admin_cluster_commands_test.go

```diff
@@ -134,3 +137,348 @@ func TestValidSearchAttributeKey(t *testing.T) {
 	assert.Error(t, visibility.ValidateSearchAttributeKey("9lives"))
 	assert.Error(t, visibility.ValidateSearchAttributeKey("tax%"))
 }
+
+func TestAdminDescribeCluster(t *testing.T) {
+	// Initialize mock controller
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	// Create mock frontend and admin clients
+	serverFrontendClient := frontend.NewMockClient(mockCtrl)
```

Comment:
I highly recommend using different mock instances in different test cases to achieve isolation.

Reply:
This one has been addressed in another PR